src/external/spglib.jl

```diff
@@ -44,21 +44,27 @@ spglib_cell(lattice, atoms) = first(spglib_cell_atommapping_(lattice, atoms))
     end
 
     # Ask spglib for symmetry operations and for irreducible mesh
-    spg_symops = spglib.get_symmetry(spglib_cell(lattice, atoms),
-                                     symprec=tol_symmetry)
+    spg_rotations    = Array{Cint}(undef, 3, 3, max_symops)
+    spg_translations = Array{Cdouble}(undef, 3, max_symops)
+    spg_positions, spg_numbers = spglib_atoms(atoms)
+    
+    spg_num_ops = ccall((:spg_get_symmetry, SPGLIB), Cint,
+       (Ptr{Cint}, Ptr{Cdouble}, Cint, Ptr{Cdouble}, Ptr{Cdouble}, Ptr{Cint}, Cint, Cdouble),
+       spg_rotations, spg_translations, Cint(max_symops), lattice, spg_positions, spg_numbers,
+       Cint(length(spg_numbers)), tol_symmetry)
+       
+    # Note: spglib_jll returns transposed rotation matrix vs the python spglib interface
 
     # If spglib does not find symmetries give an error
-    if spg_symops === nothing
-        err_message=spglib.get_error_message()
+    if spg_num_ops == 0
+        err_message = spglib_get_error_message()
         error("spglib failed to get the symmetries. Check your lattice, use a " *
               "uniform BZ mesh or disable symmetries. Spglib reported : " * err_message)
     end
 
-    Stildes = [St for St in eachslice(spg_symops["rotations"]; dims=1)]
-    τtildes = [rationalize.(τt, tol=tol_symmetry)
-               for τt in eachslice(spg_symops["translations"]; dims=1)]
-    @assert length(τtildes) == length(Stildes)
-
+    Stildes = [Mat3{Int}(spg_rotations[:, :, i])' for i in 1:spg_num_ops]
+    τtildes = [rationalize.(Vec3{Float64}(spg_translations[:, i]), tol=tol_symmetry) for i in 1:spg_num_ops]
```

Comment:
Line getting a bit long.

Reply:
No clue where to break this line without it being ugly, any suggestions?

Reply:
That's really a tough one. I think I would split it up in two expressions. There is anyway a lot happening in one line.

Reply:
What we do in other places is 
```
x = [y
     for y in z]
```
which is not so bad I thought.

I don't particularly care about line length but Michael just really hates long lines because he has a very primitive editor ;-)